spec/test_apps/app/dumpers/tests_response_dumper.rb

```diff
@@ -10,4 +10,17 @@
     get root_path
     delete url_for(controller: :tests, action: :destroy)
   end
+
+  dump 'multipart_formdata_request_with_file', status_codes: %i[no_content] do
+    mocked_image_file = Rack::Test::UploadedFile.new(
+      StringIO.new('png file data content'),
+      'image/png',
+      true,
+      original_filename: 'fake_image_file.png'
+    )
+
+    post url_for(controller: :tests, action: :submit_image), params: {
+      uploaded_image_file: mocked_image_file
+    }
```

Comment:
Fits nicely on one line:

```suggestion
    post url_for(controller: :tests, action: :submit_image), params: { uploaded_image_file:  mocked_image_file }
```